libs/openFrameworks/graphics/ofPixels.cpp

```diff
@@ -893,17 +894,17 @@ void ofPixels_<PixelType>::setImageType(ofImageType imageType){
 	if(!isAllocated() || imageType==getImageType()) return;
 	ofPixels_<PixelType> dst;
 	dst.allocate(width,height,imageType);
-	PixelType * dstPtr = &dst[0];
-	PixelType * srcPtr = &pixels[0];
-	int dstNumChannels = dst.getNumChannels();
-	int srcNumChannels = getNumChannels();
-	int diffNumChannels = 0;
+	PixelType * dstPtr = &dst[static_cast<size_t>(0)];
```

Comment:
I don't understand why casts are needed here ... are you still getting the warning now that the other bits are changed?